### PR TITLE
fix(human-app): footer and navbar width and padding adjustments

### PR DESCRIPTION
**Pull request by rozekmichal**

### CODE REVIEW

File: packages/apps/human-app/frontend/src/components/layout/drawer-menu-items/drawer-menu-items-worker.tsx [hunk 1]
```diff
@@ -9,7 +9,8 @@ import { HelpIcon, UserOutlinedIcon, WorkIcon } from '@/components/ui/icons';
 import { routerPaths } from '@/router/router-paths';
 
 export const workerDrawerTopMenuItems = (
-  addressRegistered: boolean
+  addressRegistered: boolean,
+  hasSiteKey: boolean
```
Comment (KacperKoza343) [inline packages/apps/human-app/frontend/src/components/layout/drawer-menu-items/drawer-menu-items-worker.tsx]: rename it to `labelingEnabled`

File: packages/apps/human-app/frontend/src/components/layout/protected/drawer-navigation.tsx [hunk 2]
```diff
@@ -151,54 +160,62 @@ export function DrawerNavigation({
...
+                        element.target = '_blank';
+                        document.body.appendChild(element);
+                        element.click();
+                        return;
+                      }
+                      if (link && !href) {
+                        navigate(link);
+                      }
+                    }}
                     sx={{
```
Comment (KacperKoza343) [inline packages/apps/human-app/frontend/src/components/layout/protected/drawer-navigation.tsx:189]: I think that `ListItemButton` has `selected` props, use it

File: packages/apps/human-app/frontend/src/components/layout/drawer-menu-items/drawer-menu-items-worker.tsx (hunk 1 above)
Comment (rozekmichal) [reply packages/apps/human-app/frontend/src/components/layout/drawer-menu-items/drawer-menu-items-worker.tsx]: done

File: packages/apps/human-app/frontend/src/components/layout/protected/drawer-navigation.tsx (hunk 2 above)
Comment (rozekmichal) [reply packages/apps/human-app/frontend/src/components/layout/protected/drawer-navigation.tsx:189]: done

**Pull request merged**